packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -253,6 +254,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         className={ css('ms-Dropdown-items', styles.items) }
         aria-labelledby={ id + '-label' }
         onKeyDown={ this._onZoneKeyDown }
+        role='listbox'
```

Comment:
No.

The combobox is the main control that indicates what is selected, and can pop open a list of choices.

The list of choices that the combobox shows is a listbox (with options).

This is the same structure used in the example I linked.